### PR TITLE
Adds style for input focus

### PR DESCRIPTION
**Pull request by jchristo4**
image
image

### CODE REVIEW

File: app/assets/stylesheets/sul-variables.scss [hunk 1]
```diff
@@ -15,6 +16,7 @@ $sul-footer-bg-color: $beige-10-percent;
 $sul-h1-font-color: $gray;
 $sul-h2-border-color: $beige-30-percent;
 $sul-h2-font-color: $cardinal-red;
+$sul-form-input-focus: $mostly-pure-orange;
```
Comment (jkeck) [inline app/assets/stylesheets/sul-variables.scss]: It doesn't look like the `$sul-form-input-focus` is used anywhere (I guess `scss-lint` doesn't catch that).

File: app/assets/stylesheets/modules/forms.scss [hunk 2]
```diff
@@ -1,4 +1,13 @@
 .form-group {
+  a.btn:focus,
```
Comment (jkeck) [inline app/assets/stylesheets/modules/forms.scss]: Should these rules apply outside of the `.form-group` class or should they be global to any input/button regardless of where it shows up?

File: app/assets/stylesheets/sul-variables.scss (hunk 1 above)
Comment (jchristo4) [inline app/assets/stylesheets/sul-variables.scss]: @jkeck Fixed

File: app/assets/stylesheets/modules/forms.scss (hunk 2 above)
Comment (jchristo4) [inline app/assets/stylesheets/modules/forms.scss]: @jkeck Added scss-mixin for using the outline style outside `.form-group` elements

**Comment by jchristo4** [comment]
Had to add `!important` in mixin because bootstrap resets global outline styles

**Comment by jkeck** [comment]
@jchristo4 it looks like Bootstrap stores the border focus color for inputs in the variable `$input-border-focus`.  Perhaps it's possible we can override that variable w/ our color variable, and then add the mixin (or straight css) to the link focus.  Let me know if going that route doesn't work and we can :ship: the `!important`.

**Pull request merged**